### ISSUE TITLE
zsh prints raw escape sequence in prompt: \[\e]0;\w\a\]\n\[\e[32m\]\u@\h

### ISSUE THREAD

**Issue by Maximus5** [msys2/MSYS2-packages#1285]
Just installed fresh Msys2 from scratch (msys2-x86_64-20180531.exe), updated all packages and installed zsh via `pacman -S zsh`.
Started zsh from bash prompt (MSYS2 msys), using default configuration (in zsh first time startup script) and zsh weird prompt appeared

```
max@maxpc MSYS ~
$ zsh
\[\e]0;\w\a\]\n\[\e[32m\]\u@\h \[\e[35m\]$MSYSTEM\[\e[0m\] \[\e[33m\]\w\[\e[0m\]\n\$
```

**Comment by Alexpux** [comment]
@Maximus5 I'm not familar with zsh, so dont know what happened. Did you restart MSYS2 after packages upgrade?

**Comment by Alexpux** [comment]
@Maximus5 just tried myself and all works ok
````
Alexx@PC MSYS /repo/mingw32/scripts
$ zsh
This is the Z Shell configuration function for new users,
zsh-newuser-install.
You are seeing this message because you have no zsh startup files
(the files .zshenv, .zprofile, .zshrc, .zlogin in the directory
~).  This function can help you with a few settings that should
make your use of the shell easier.

You can:

(q)  Quit and do nothing.  The function will be run again next time.

(0)  Exit, creating the file ~/.zshrc containing just a comment.
     That will prevent this function being run again.

(1)  Continue to the main menu.

--- Type one of the keys in parentheses --- 0
Alexx@PC /repo/mingw32/scripts %
Alexx@PC /repo/mingw32/scripts % ls
1.txt  add-packages.sh  sync-packages.sh
Alexx@PC /repo/mingw32/scripts %
````

**Comment by Maximus5** [comment]
Yes, of course, I have restarted MSYS2 (mintty) several times.
It looks like zsh adopts `PS1` from bashrc and does that improperly (on my PC).
I tried both 32-bit and 64-bit installations of Msys2 - both the same.

Strange, but if I run `zsh -l` it does not take care of PS1

```
max@maxpc MSYS ~
$ zsh -l
(max@maxpc)[1] ~ % 
```

**Comment by Maximus5** [comment]
To clarify.
I do not suffer from the problem, but that may be some configuration issue appeared after fresh install.

**Comment by DrazharLn** [comment]
The issue is that PS1 is exported by /etc/profile. If the zsh config doesn't set one explicitly then zsh's sane default won't be used.

The solution here, IMHO, is to not export PS1. PS1 is often uses a shell specific syntax and should not be exported to other programs. You can set it in your .rc file instead, and indeed /etc/bash.bashrc does set it.

This also fixes an issue when switching between shells (e.g. run bash then zsh, then bash). If you do this, the PS1 value defined by zsh will be exported to bash (and the global bashrc won't overwrite it).

**Comment by DrazharLn** [comment]
`zsh -l` sources /etc/zsh/zprofile which sources /etc/profile which sets PS1 to an appropriate value for zsh, so that makes sense.

@Alexpux I don't know why you're not experiencing the bug, but the prompt you're getting is not the default msys2 zsh prompt (set in /etc/profile), so maybe your previous shell's PS1 is compatible or you have some extra global config file that's setting PS1.

**Comment by Alexpux** [comment]
@DrazharLn I'm not use zsh myself

**Comment by DrazharLn** [comment]
Will you accept a PR that stops /etc/profile from exporting PS1?

**Comment by Alexpux** [comment]
Create PR and I will review

**Comment by n1kk** [comment]
I have the same issue.
![image](https://user-images.githubusercontent.com/5442968/59568280-e5f49f00-9078-11e9-971b-0390c99f58ba.png)

Was there any solution to this?

**Comment by hiroakit** [comment]
@Alexpux @lazka 
I would like to talk this issue on #1624, because it also happen in the latest environment.
Thus, would you mind closing this one?

**Comment by hiroakit** [comment]
This issue has been fixed in #1903. 
Please update to `msys/zsh 5.8-3` and check it out.
